apps/gallery/stories/composites/wui-card-select.stories.ts

```diff
@@ -0,0 +1,29 @@
+import type { Meta } from '@storybook/web-components'
+import '@web3modal/ui/src/composites/wui-card-select'
+import '../../components/gallery-container'
+import type { WuiCardSelect } from '@web3modal/ui/src/composites/wui-card-select'
+import { html } from 'lit'
+
+type Component = Meta<WuiCardSelect>
+
+export default {
+  title: 'Composites/wui-card-select',
+  args: {
+    imageSrc:
```

Comment:
Could reuse that `walletImagePresets[0].src` trick from last pr here as well, maybe there are other places?